xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-macro/src/main/resources/ApplicationResources.properties

```diff
@@ -91,8 +91,8 @@ livedata.pagination.last=Last page
 
 livedata.action.refresh=Refresh
 livedata.action.addEntry=Add entry
-livedata.action.reorder.hint=Drag to reorder
-livedata.action.resizeColumn.hint=Drag to resize. Double click to reset.
+livedata.action.columnName.hint=Click to sort the entries based on this property. Drag or use arrow keys to reorder.
```

Comment:
I think we should have two titles: One when the column is sortable, and another one when it isn't.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2475/commits/ff4685a27336fd8a3a49d875055980339f354d42 :+1: